lib/WWW/WebKit2/MouseInput.pm

```diff
@@ -2,13 +2,25 @@ package WWW::WebKit2::MouseInput;
 
 use Moose::Role;
 use Carp qw(carp croak);
+use File::Basename qw(dirname);
+use File::Slurper qw(read_text);
 
 has event_send_delay => (
     is  => 'rw',
     isa => 'Int',
     default => 0, # ms
 );
 
+has 'drag_and_drop_sim' => (
+    is      => 'ro',
+    isa     => 'Str',
+    lazy    => 1,
+    default => sub {
+        # https://github.com/Photonios/JS-DragAndDrop-Simulator
+        return read_text(dirname(__FILE__ ) . '/../../../resources/javascript/dndsim.js') . '1;';
```

Comment:
This will only work on a git checkout of the WWW::WebKit2 module. It won't work when the module is installed.